include/Clip.h

```diff
@@ -159,6 +175,7 @@ public slots:
 	BoolModel m_mutedModel;
 	BoolModel m_soloModel;
 	bool m_autoResize;
+	bool m_hasBeenResized;
```

Comment:
```suggestion
	bool m_hasBeenResized = false;
```
This member is uninitialized which could cause problems when using the return value of `getHasBeenResized`. I like to initialize members like this right where they are defined, but you can also do it in the constructor(s) - it's just more work that way and you could forget.